File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/O. Calculator/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/O

#include <iostream>
using namespace std;

int main()
{
    int a, b;
    char s;
    cin >> a >> s >> b;
    switch (s)
    {
    case '+':
        cout << a+b << endl; return 0;
    case '-':
        cout << a-b << endl; return 0;
    case '*':
        cout << a*b << endl; return 0; 
    default:
        cout << a/b << endl; return 0; 
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #1 (Data type - Conditions)/O. Calculator/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/O

using System;

namespace Codeforces.Assuit.Sheet1
{
    class ProblemO
    {
        static void Main(string[] args)
        {
            char[] ops = { '+', '-', '*', '/' };
            string input = Console.ReadLine();
            string[] parts = input.Split(
                ops,
                StringSplitOptions.RemoveEmptyEntries);

            int a = int.Parse(parts[0]);
            int b = int.Parse(parts[1]);
            char op = input[input.IndexOfAny(ops)];
            switch (op)
            {
                case '+':
                    Console.WriteLine(a + b);
                    return;
                case '-':
                    Console.WriteLine(a - b);
                    return;
                case '*':
                    Console.WriteLine(a * b);
                    return;
                case '/':
                    Console.WriteLine(a / b);
                    return;

            }
        }
    }
}
